test/save-in-progress/01-sip-load-fail.e2e.spec.js

```diff
@@ -37,16 +37,19 @@ module.exports = E2eHelpers.createE2eTest(
 
     // fail to find in progress form
     client
+      .url(url)
+      .waitForElementVisible('body', Timeouts.normal)
       .mockData({
         path: '/v0/in_progress_forms/1010ez',
         verb: 'get',
         value: {},
         status: 404
       }, token)
+      .pause(500)
```

Comment:
Would it be better to `.waitForElementVisible` instead of just pausing?

Reply:
`.waitForElementVisible` didn't work. There is some sort of timing issue maybe with how event handlers are being setup by react where clicking on the link doesn't do anything until a few ms after the element has rendered on the page.

Reply:
If that's the case, it's surprising that it's only an issue here and not more widespread.

I think it would be safer to wait for the element to be visible/present, then pause, then click. When we've added pauses to wait for something to render, it's usually bitten us later.

Reply:
Yeah, that's really odd. :confused:

Reply:
Sure I don't mind adding a `.waitFor..` method. I'll also try to look into more if this is something odd selenium is doing.

Reply:
I  think I have a fix for the `.pauses()`. It turns out this is an issue with the element to click being offscreen (not sure if this issue is with Selenium or Chrome). Instead of pausing, there's a nightwatch method called `.getLocationInView` that makes us scroll to the element, which we can then successfully click on.